### ISSUE TITLE
Test for changes in the number of speeches

### ISSUE THREAD

**Issue by MansMeg** [swerik-project/riksdagen-records#49]
We should test that the number of speeches doesn't change between versions of the coprus. 

Or rather, if there is a change, we need to update the unit test simultaneously to be very defensive.

**Comment by fredrik1984** [comment]
Good idea. The number of speeches is likely to change if the segmentation of speeches improves. For example, better segmentation might lead to more annotated speeches (from one incorrectly annotated speech to several), or fewer speeches.

**Comment by MansMeg** [comment]
Yes. I think that is reasonable. But then we will change the unit test at the same time. But if we don't, e.g. change the note/seg classifications, the no of speeches should remain the same. Right?

**Comment by BobBorges** [comment]
The number of speeches should stay the same --- if we accept @ninpnin 's definition so long as no intros change.

**Comment by ninpnin** [comment]
I would enforce a Jaccard coefficient limit (e.g. J >= 0.95) from the current speech IDs.
